### PR TITLE
Logo

### PR DESCRIPTION
**Pull request by chriswhong**
Adds a 32x32 qri logo to the top left of the dataset view
Qri_Desktop_and_Comparing_master___logo_·_qri-io_desktop

### CODE REVIEW

File: app/containers/DatasetContainer.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { Resizable } from '../components/resizable'
 import DatasetSidebar from '../components/DatasetSidebar'
 
 const defaultSidebarWidth = 250
+const logo = require('../assets/qri-blob-logo-tiny.png') //eslint-disable-line
 
 export default class DatasetContainer extends React.Component<{}, { showDatasetList: boolean, sidebarWidth: number }> {
```
Comment (ramfox) [inline app/containers/DatasetContainer.tsx]: Didn't we change this to `Dataset` and move it to `/components/Dataset`?

File: app/scss/_dataset-container.scss [hunk 2]
```diff
@@ -22,7 +22,7 @@
     display: inline-block;
```
Comment (ramfox) [inline app/scss/_dataset-container.scss]: Mentioned this in the other pr, but this file should be `_dataset.scss`, and the old `_dataset.scss` should be `_dataset-old.scss`

File: package.json [hunk 3]
```diff
@@ -137,7 +137,7 @@
     "electron-devtools-installer": "2.2.4",
     "enzyme": "3.10.0",
     "enzyme-adapter-react-16": "1.14.0",
-    "eslint": "6.0.1",
+    "eslint": "^6.0.1",
```
Comment (ramfox) [inline package.json]: As a convention, we put exact versions, should go back to ` "eslint": "6.0.1",`

File: package.json [hunk 4]
```diff
@@ -137,7 +137,7 @@
...
     "eslint-config-standard-with-typescript": "7.1.0",
     "eslint-loader": "^2.2.1",
```
Comment (ramfox) [inline package.json]: Same comment as above: should be changed to `"eslint-loader": "2.2.1",`

File: app/containers/DatasetContainer.tsx (hunk 1 above)
Comment (chriswhong) [reply app/containers/DatasetContainer.tsx]: I'll rename it again now.

File: package.json (hunk 3 above)
Comment (chriswhong) [reply package.json]: fixed

File: package.json (hunk 4 above)
Comment (chriswhong) [reply package.json]: fixed

**Pull request merged**